setup.py

```diff
@@ -35,7 +35,7 @@
         "opt_einsum",
         "scipy",
         # pinned for compatibility with strawberry fields
-        "antlr4-python3-runtime==4.8",
+        "antlr4-python3-runtime==4.9",
```

Comment:
Can we pin to 4.9.2 ?

Reply:
My understanding is that this is a looser but sufficient constraint. If another library upgrades from 4.9.2 to 4.9.3, then we won't have to make a code change here

Reply:
```suggestion
        "antlr4-python3-runtime==4.9.2",
```